app/scripts/modules/amazon/src/loadBalancer/AmazonLoadBalancerClusterContainer.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import autoBindMethods from 'class-autobind-decorator';
+
+import { ILoadBalancerClusterContainerProps, LoadBalancerClusterContainer } from '@spinnaker/core'
+
+import { IAmazonApplicationLoadBalancer } from '../domain/IAmazonLoadBalancer';
+import { TargetGroup } from './TargetGroup';
+
+interface IState {
+}
+
+@autoBindMethods
+export class AmazonLoadBalancerClusterContainer extends React.Component<ILoadBalancerClusterContainerProps, IState> {
+  constructor(props: ILoadBalancerClusterContainerProps) {
+    super(props);
+    this.state = {
+    };
```

Comment:
note: this could also be written without the constructor and a property initializer
```js
class Foo {
  public state: IState = {};
  ...
}
```